apps/publishers/signals.py

```
@@ -0,0 +1,9 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from django.core.cache import cache
+from apps.publishers.models import Domain
+
+
+@receiver(post_save, sender=Domain)
+def clear_domain_cache(sender, instance, **kwargs):
```

Ruff (0.15.2)

[warning] Unused function argument: `sender`

(ARG001)